apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/ItemService.java

```diff
@@ -61,8 +65,18 @@ public void updateConfigItemByText(NamespaceTextModel model) {
     long namespaceId = model.getNamespaceId();
     String configText = model.getConfigText();
 
-    ConfigTextResolver resolver =
-        model.getFormat() == ConfigFileFormat.Properties ? propertyResolver : fileTextResolver;
+    ConfigTextResolver resolver = fileTextResolver;
+    switch (model.getFormat()) {
+      case Properties:
+        resolver = propertyResolver;
+        break;
+      case YML:
+      case YAML:
+        resolver = yamlResolver;
```

Comment:
I think `YML` can use this `yamlResolver`, too?

Reply:
yes，they are essentially identical except the extension name

Reply:
> yes，they are essentially identical except the extension name

yeah, i mean that you can add `      case YML: resolver = yamlResolver;`  here.

Reply:
There is no need to do so, we use fall back trick here, i.e. no break statement after YML, anyway, adding it explicitly may be a better practice.